modules/ROOT/pages/dataweave-cookbook.adoc

```diff
@@ -84,47 +84,35 @@ This section links to examples that demonstrate common data extraction and trans
+| xref:dataweave-cookbook-use-constant-directives.adoc[Use Constant Directives]
 | Defines a `var` in the DataWeave header that builds URLs from constant directives using `++` to concatenate strings.
 
-| xref:dataweave-cookbook-parse-dates.adoc[Parsing Dates]
+| xref:dataweave-cookbook-parse-dates.adoc[Parse Dates]
 | Defines a function that normalizes conflicting date formats into a single, common format. Uses `mapObject`, `replace` and `as` to coerce a data type.
 
-| xref:dataweave-cookbook-add-and-subtract-time.adoc[Adding and Subtracting Dates]
+| xref:dataweave-cookbook-add-and-subtract-time.adoc[Add and Subtract Dates]
 | Performs multiple math operations combining different types related to date and time.
 
-| xref:dataweave-cookbook-conditional-list-reduction-via-function.adoc[Conditionally Reduce a List Via a Function]
+| xref:dataweave-cookbook-conditional-list-reduction-via-function.adoc[Conditionally Reduce a List through a Function]
 | Defines a function that reduces a list of elements into one field. It then calls this function only when a certain field has values. Uses `map`, `reduce`, `splitBy`, `if` and `++` to append to an array.
 
-| xref:dataweave-cookbook-map-based-on-an-external-definition.adoc[Dynamic Mapping Based On a Definition]
+| xref:dataweave-cookbook-map-based-on-an-external-definition.adoc[Dynamic Map Based On a Definition]
 | Transforms the payload according to definitions sent in a variable. It defines a function that applies the logic defined in the variable, using `map` and `default`.
 
-|  xref:dataweave-cookbook-pass-functions-as-arguments.adoc[Passing Functions as Arguments]
+|  xref:dataweave-cookbook-pass-functions-as-arguments.adoc[Pass Functions as Arguments]
 | Defines a function that expects to receive two inputs: a function to apply and an element to apply it on. The function is also recursively applied to the element's children. It uses `mapObject` `lower` `is` `if/else`.
 
 
 |===
 
-////
-THESE DON'T WORK
-|  link:dataweave-configure-csv-reader[]
-
-|  link:dataweave-cookbook-create-mule-config[Creating a Mule Configuration]
-
-|  link:dataweave-cookbook-create-mule-pom[Creating a Mule POM]
-
-
-////
-
-
 
 
 == See Also
 
 ////
-* To step through basic DataWeave use cases and exercises in actual Mule applications, see the xref:dataweave-quickstart.adoc[DataWeave Quickstart Guide].
+* To step through basic DataWeave use cases and exercises in actual Mule applications, see the xref:dataweave-quickstart.adoc[DataWeave Tutorial].
 ////
 * xref:dataweave.adoc[DataWeave Language]
-* xref:dataweave-language-introduction.adoc[DataWeave Language Introduction]
-* xref:dw-functions.adoc[DataWeave Functions]
-* xref:mule-concepts.adoc[Mule Concepts]
\ No newline at end of file
+* xref:dataweave-language-introduction.adoc[DataWeave Scripts]
+* xref:dw-functions.adoc[DataWeave Reference]
+* xref:mule-concepts.adoc[Key Concepts]
```

Comment:
Key Mule Concepts   ?

Reply:
Looks like the title of mule-concepts.adoc is "Key Concepts"